setup/gcp/installer/util/gcloud.py

```diff
@@ -626,6 +631,14 @@ def get_vm_instance_info(instance_name, try_match=False):
         str: Zone of the forseti server application, default to 'us-central1-c'
         str: Name of the forseti server instance
     """
+    def _ping_compute_instance():
+        """Check compute instance status.
```

Comment:
nit: Closing """ should be on the same line.

Reply:
Updated.